### PR TITLE
[FEATURE] InTimezone select modificator

### PR DESCRIPTION
**Pull request by GDVFox**

### CODE REVIEW

File: select_builder.go [hunk 1]
```diff
@@ -84,13 +91,45 @@ func (tx *Tx) SelectBySql(query string, value ...interface{}) SelectBuilder {
 	}
 }
 
+func (b *selectBuilder) changeTimezone(value reflect.Value) {
+	if b.timezone == nil {
```
Comment (DoubleDi) [inline select_builder.go]: can we put this in the upper func, to make reflect.ValueOf calls?

File: select_builder.go [hunk 2]
```diff
@@ -84,13 +91,45 @@ func (tx *Tx) SelectBySql(query string, value ...interface{}) SelectBuilder {
...
+		return
+	}
+
+	v, t := extractOriginal(value)
```
Comment (DoubleDi) [inline select_builder.go:95]: and what happens if you get a nil pointer or interface here?

Comment (GDVFox) [reply select_builder.go:95]: Nothing.

File: select_builder.go (hunk 1 above)
Comment (GDVFox) [reply select_builder.go]: ok

**Pull request merged**